Source/BugsnagClient.m

```diff
@@ -84,6 +84,10 @@
 static NSUInteger handledCount;
 static NSUInteger unhandledCount;
 static bool hasRecordedSessions;
+#if TARGET_IPHONE_SIMULATOR || TARGET_OS_IPHONE
+// The previous device orientation - iOS only
+static NSString *lastOrientation = NULL;
```

Comment:
Can this be declared as a property instead? This would make it easier to get to a point where it'd be possible to instantiate multiple `BugsnagClient` objects, and also simplify any future extraction of this logic to a separate class.

Reply:
I'm assuming you mean class property?  Yes, that makes sense.

Reply:
Yep, that's correct